apkid/rules/dex/obfuscators.yara

```diff
@@ -187,3 +187,23 @@ rule allatori_demo : obfuscator
   condition:
     $s and is_dex
 }
+
+rule aamo : obfuscator
+{
+  meta:
+    description = "AAMO"
+    author = "P0r0"
+    url = "https://github.com/necst/aamo"
+    backup_url = "https://github.com/P0r0/aamo"
```

Comment:
Why the backup_url?

Reply:
i forked it just in case the repo is deleted. Even though it has been forked 37 times already :D...I can remove that tag if useless

Reply:
We can keep it if you think there's a risk of the original being taken down. Is that the case?